src/Monolog/Handler/SlackHandler.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace MatchBot\Monolog\Handler;
+
+use MatchBot\Application\SlackChannelChatterFactory;
+use Monolog\Handler\HandlerInterface;
+use Monolog\Logger;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Notifier\Bridge\Slack\Block\SlackHeaderBlock;
+use Symfony\Component\Notifier\Bridge\Slack\Block\SlackSectionBlock;
+use Symfony\Component\Notifier\Bridge\Slack\SlackOptions;
+use Symfony\Component\Notifier\ChatterInterface;
+use Symfony\Component\Notifier\Message\ChatMessage;
+
+/**
+ * Sends error messages to our slack alarm channel so we deal with the errors without unnecessary delay.
+ *
+ * @phpstan-import-type Record from \Monolog\Logger
+ * @phpstan-import-type Level from \Monolog\Logger
+ * @psalm-import-type Record from \Monolog\Logger
+ * @psalm-import-type Level from \Monolog\Logger
+ *
+ */
+class SlackHandler implements HandlerInterface
+{
+    public function __construct(private ChatterInterface $slackConnction)
+    {
+
+    }
+
+    public function isHandling(array $record): bool
+    {
+        return $record['level'] >= Logger::ERROR;
+    }
+
+    /**
+     * @psalm-suppress PossiblyUnusedReturnValue - used by Monolog.
+     */
+    public function handle(array $record): bool
+    {
+        if ($record['level'] < Logger::ERROR) {
+            return false; // allows another handle to handle this.
+         }
+
+        $message = $record['message'];
+        $levelName = $record['level_name'];
```

Comment:
We could also serialise `$record['context']` and include it in the slack message - but I thought maybe better to leave that out. If we want those details we can find them in the logs elsewhere, and we might choose to put more sensitive info in the context than we do in the message sometimes. Although we probably just avoid putting sensitive info in either and I think we don't actually use context much if at all.

Reply:
See docs for Record type at https://github.com/Seldaek/monolog/blob/2.x/doc/message-structure.md